libqtile/backend/x11/drawer.py

```diff
@@ -83,7 +83,11 @@ def _create_gc(self):
 
     def _free_gc(self):
         if self._gc is not None:
-            self.qtile.core.conn.conn.core.FreeGC(self._gc)
+            try:
+                self.qtile.core.conn.conn.core.FreeGC(self._gc)
+            except xcffib.ConnectionException:
+                # Qtile got disconnected from the X server
+                pass
```

Comment:
Might be overkill but you use this `try... except...` block multiple times. We could create a context manager wrapper so you could do something like:
``` python
with skip_on_connection_exception():
    self.qtile.core.conn.conn.core.FreeGC(self._gc)
```

Reply:
Yeah I like it. DRY. Will do.

Reply:
Nice. Didn't know about `contextlib.suppress`. Exactly what's needed.